packages/faustwp-cli/utils/log.ts

```diff
@@ -1,15 +1,19 @@
 import chalk from 'chalk';
 
 export const log = (
-  logLevel: 'notice' | 'error',
+  logLevel: 'info' | 'warn' | 'error',
   message: string,
   ...args: any
 ) => {
-  let logLevelMessage = chalk.whiteBright('faust: ');
+  let logLevelMessage = chalk.whiteBright('faustwp: ');
```

Comment:
Using `faustwp` here to match the CLI's bin name

Reply:
Since this is human readable, I think we'll need to go with `Faust`